packages/graphql-yoga/package.json

```diff
@@ -71,7 +71,7 @@
     "@repeaterjs/repeater": "^3.0.4",
     "@types/node": "18.15.11",
     "graphql": "^16.0.1",
-    "graphql-http": "^1.7.2",
+    "graphql-http": "file:../../graphql-http-1.18.0-rc.1.tgz",
```

Comment:
👀

Reply:
yeah, graphql-http doesn't have alpha releases for PRs - so I just packed with https://github.com/graphql/graphql-http/pull/76 and installed :D